README.md

```diff
@@ -59,6 +61,22 @@ To run a specific set of tests, use the `-run` flag and specify a regex pattern
 $ task test -- -run 'TestAcc_Project*'
 ```
 
+## Running the provider locally
+
+Set up a local override on your machine
+
+```sh
+$ task install
+```
+
+Build the provider
+
+```sh
+$ task build
+```
+
+Create a `main.tf` file on your machine and use the [terraform cli](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform) to test
```

Comment:
This is nice, but maybe remove/clean up the section that's already there? 
<img width="1451" alt="Screenshot 2023-11-23 at 15 33 00" src="https://github.com/vercel/terraform-provider-vercel/assets/8166969/0803cc57-8c03-42a0-a277-9ebeede9f7f9">

Reply:
oops I hadn't read that fully, I'll combine them